### PR TITLE
Make ECoG and sEEG size defaults more realistic

### PR DESCRIPTION
**Pull request by libertyh**
#### Reference issue
Fixes issue #12472 

#### What does this implement/fix?
Makes a minor change to the ECoG and sEEG default electrode sizes in `defaults.py` so they are more realistic (`2e-3` instead of `5e-3`). This makes it so sEEG and ECoG electrodes do not overlap one another and better reflect the actual size of electrodes in real devices.

#### Additional information
The original size was `5e-3` and new size is `2e-3`. This reflects a 2mm contact size, which is closer to the typical 0.8-2.5 mm contact sizes for ECoG and sEEG. This also means that electrodes with spacing <5mm (which is quite typical) will not overlap in the visualizations.

### CODE REVIEW

**Comment by agramfort** [comment]
@alexrockhill ok for you?

**Comment by libertyh** [comment]
> @alexrockhill ok for you?

If not, I can also go back to implementation (2) in issue #12472 and just have it as an option to pass electrode size instead. Thanks for any thoughts you have!

**Comment by agramfort** [comment]
both would work for me but maybe a better default is enough?

**Comment by alexrockhill** [comment]
Looks like a good default size to me! Glad you're using this @libertyh!

**Comment by drammock** [comment]
@libertyh please add a new file `doc/changes/devel/12474.bugfix.rst` with a changelog entry. View other files in that folder for formatting/content inspiration. (I'm suggesting calling this a bugfix and not enhancement based on "unrealistically large to the point of visual overlap").

File: doc/changes/devel/12473.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+- Changed default ECoG and sEEG electrode sizes in `mne/defaults.py` to better reflect real world sizes, by `Liberty Hamilton`_
```
Comment (drammock) [inline doc/changes/devel/12473.bugfix.rst:1]:
```suggestion
- Changed default ECoG and sEEG electrode sizes to better reflect real world sizes, by `Liberty Hamilton`_
```

The single ticks around `mne/defaults.py` attempt to make it an API crossref; double-backticks would work, but end users don't really need to know which file the change was in so let's just remove the file reference

Comment (libertyh) [reply doc/changes/devel/12473.bugfix.rst:1]: This is named according to my old PR, that's why it's failing! I'll fix the name. Thanks!

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
test failures are unrelated --- we're testing against pre-release of NumPy 2.0 which has some API changes, and a dependency (pandas) hasn't yet accomodated. So I'll go ahead and merge; thanks @libertyh!

**Pull request merged**